frontend/src/component/GalaxyLogin.jsx

```
@@ -0,0 +1,109 @@
+            <input
+              type={showPassword ? "text" : "password"}
+              name="password"
+              placeholder="Password"
+              value={user.password}
+              onChange={userDataChange}
+              className="w-full p-3 pr-10 rounded-md bg-gray-800 text-white border border-gray-600 focus:ring-2 focus:ring-blue-500"
+            />
+            <span
+              className="absolute top-1/2 right-3 transform -translate-y-1/2 text-gray-400 cursor-pointer"
+              onClick={() => setShowPassword((prev) => !prev)}
+            >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)